projects/novo-examples/src/utils/drag-drop/drag-drop/drag-drop-example.ts

```diff
@@ -17,23 +17,23 @@ interface DemoButtonObject {
 })
 export class DragDropExample implements OnChanges {
   objects: DemoButtonObject[] = [
-    { name: 'Object 1', btnText: 'Button 1', bgColor: 'red' },
-    { name: 'Object 2', btnText: 'Button 2', bgColor: 'blue' },
-    { name: 'Object 3', btnText: 'Button 3', bgColor: 'green' },
-    { name: 'Object 4', btnText: 'Button 4', bgColor: 'yellow' },
-    { name: 'Object 5', btnText: 'Button 5', bgColor: 'wheat' },
-    { name: 'Object 6', btnText: 'Button 6', bgColor: 'purple' },
-    { name: 'Object 7', btnText: 'Button 7', bgColor: 'teal' }
+    { name: 'Object 1', headerText: 'Item 1', bgColor: 'rgba(255,0,0,0.3)' },
```

Comment:
Would it make sense to use the standard bullhorn colors here?

Reply:
At first, I was preferring something more muted to make sure they'd all be readable with black text. This came up in standup though, and since the word "Title" is hardly important, I think those do work out better. I've just updated it.